codalab/worker/file_util.py

```diff
@@ -148,7 +149,7 @@ def zip_directory(
             raise IOError(e.output)
 
 
-def unzip_directory(fileobj_or_name, directory_path, force=False):
+def unzip_directory(fileobj: IO[bytes], directory_path: str, force: bool = False):
```

Comment:
The only usage of `unzip_directory` now passes in a fileobj as the first argument, not a string.